app/views/items/show.html.erb

```diff
@@ -6,22 +6,25 @@
 <h4><%= @headline %></h4>
 <ul id="review_list">
   <% if @reviews.empty? %>
-    <li>No reviews yet!</li>
+  <li>No reviews yet!</li>
   <% else %>
-    <% @reviews.each do |review| %>
-      <li>Rating: <%= review.rating %> - <%= review.description %></li>
-    <% end %>
+  <% @reviews.each do |review| %>
+  <li>Rating: <%= review.rating %> - <%= review.description %></li>
+  <% end %>
   <% end %>
 </ul>
+<p>
+  <%= paginate @reviews %>
+</p>
 
 <h4>We'd love your review of the <%= @item.title %>!</h4>
 
 <%= form_for [@item, @review] do |f| %>
-  <%= f.label :rating, "Rating:" %>
-  <%= f.select :rating, options_for_select(@rating_options) %>
+<%= f.label :rating, "Rating:" %>
+<%= f.select :rating, options_for_select(@rating_options) %>
 
-  <%= f.label :description, "Your Review:" %>
-  <%= f.text_area :description %>
+<%= f.label :description, "Your Review:" %>
+<%= f.text_area :description %>
 
-  <%= f.submit "Submit" %>
+<%= f.submit "Submit" %>
```

Comment:
Why are the indentations gone? Anything nested inside another element should be indented.